library/Kubernetes/Web/PodListItem.php

```diff
@@ -102,7 +102,8 @@ protected function assembleFooter(BaseHtmlElement $footer): void
                 $this->item->restart_policy
             ),
             new HorizontalKeyValue(
-                (new Icon('life-ring'))->addAttributes(['title' => $this->translate('Quality of Service')]),
+                (new Icon(self::QOS_ICONS[$this->item->qos]))
```

Comment:
I would use `static` as the const is not `final`, but this is just a side note as I don't think we will ever extend `PodListItem` and overwrite the icons.